src/main.tsx

```diff
@@ -1,10 +1,92 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.tsx";
-// import './index.css'
+import { ThemeProvider, createTheme } from "@mui/material";
+
+import "@fontsource/chakra-petch";
+
+const theme = createTheme({
+  palette: {
+    background: {
+      paper: "#fff",
+    },
+    text: {
+      primary: "#000",
+      secondary: "#46505A",
+    },
+    action: {
+      active: "#000",
+    },
+  },
+  typography: {
+    h2: {
+      fontSize: "2rem",
+      fontWeight: 600,
+      fontFamily: "Chakra Petch, sans-serif",
+    },
+    h3: {
+      fontSize: "1.5rem",
+      fontWeight: 600,
+      fontFamily: "Chakra Petch, sans-serif",
+    },
+    h4: {
+      fontSize: "1.2rem",
+      fontWeight: 600,
+      fontFamily: "Chakra Petch, sans-serif",
```

Comment:
maybe just globaly set font family for all variants?

<img width="654" alt="Screenshot 2024-04-28 at 14 23 18" src="https://github.com/proga-react-practice/xi-sofiia-stanishevska/assets/103277184/85d4bb5c-caf4-41dd-8308-06f8469c8294">

Reply:
✅